src/python/python-kdmp.h

```diff
@@ -4,12 +4,6 @@
 #include "kdmp-parser.h"
 #include <Python.h>
 
-#if PY_MINOR_VERSION >= 8

```

Comment:
@masthoon Did I get this change right? Sounds like in py39 they fixed some stuff regarding the structure layout - do you mind having a quick look please?

Cheers

Reply:
@masthoon ping on this :-D